data/campaigns/Winds_of_Fate/units/elementals/Storm_Daemon.cfg

```diff
@@ -0,0 +1,172 @@
+#textdomain wesnoth-wof
+
+[unit_type]
+    id=Storm Daemon
+    name= _ "Storm Daemon"
+    race=elementals
+    image="units/elementals/storm-1.png"
+    description= _ "There are many tales of the strange storms which roam the Great Ocean and batter its shores. A number tell of twisting thunderheads moving against the wind in pursuit of ill–fated ships. These immensely powerful entities are most commonly known as Storm Daemons. Among learned magi, it is understood they are a kind of wisp, like those sighted drifting over the outlying marshes or glimmering faintly in the deep woods. Whatever else may be known of Storm Daemons is kept a secret of the elves. And perhaps also the silver order of magi, which has long studied such phenomena."
+
+    # Recruiting
+    usage=mixed fighter
+    cost=100
+
+    # Leveling
+    level=5
+    experience=300
```

Comment:
should be `experience=250` for a level 5 unit.
level 6 have 300 max XP for AMLA.

Reply:
Good catch, updated.